src/modules/main/navbar/Navbar.tsx

```diff
@@ -35,28 +40,30 @@ const Navbar = ({ basePath }: Navbar.Props) => {
 
   return (
     <nav className={csx.navbar}>
-      <div className={csx.breadcrumbs}>
-        {breadcrumbs.map((breadcrumb, idx) => (
-          <React.Fragment key={idx}>
-            <div className={csx.breadcrumb}>
-              <Link to={getLinkPath(idx, breadcrumbs, basePath)}>{breadcrumb}</Link>
-            </div>
-            <div className={csx.breadcrumbDivider}>{'>'}</div>
-          </React.Fragment>
-        ))}
-      </div>
+      {width > TABLET_WIDTH && (
+        <div className={csx.breadcrumbs}>
+          {breadcrumbs.map((breadcrumb, idx) => (
+            <React.Fragment key={idx}>
+              <div className={csx.breadcrumb}>
+                <Link to={getLinkPath(idx, breadcrumbs, basePath)}>{breadcrumb}</Link>
+              </div>
+              <div className={csx.breadcrumbDivider}>{'>'}</div>
+            </React.Fragment>
+          ))}
+        </div>
+      )}
 
       {isTemplatesRoute && (
         <Guard.Protected>
           <>
             <NavLink replace to="/app/templates/management">
-              <Button>
+              <Button variant={width > 660 ? 'default' : 'icon'}>
```

Comment:
Please instead of changing attributes - add second button which will be displayed for 600 width with predefined attributes and second one for more than 600